app/models/manageiq/providers/redhat/infra_manager/provision/state_machine.rb

```diff
@@ -24,7 +24,7 @@ def poll_clone_complete
 
     if clone_complete?
       phase_context.delete(:clone_task_ref)
-      EmsRefresh.queue_refresh(dest_cluster.ext_management_system)
+      # Full refresh was removed from here because it is expensive on larger envs
```

Comment:
Let's add a statement about what the expected behavior is just to be clear.  Something like, "Identifying the newly created VM in the database relies on eventing."